src/mcp_atlassian/test_mcp_atlassian.py

```diff
@@ -0,0 +1,180 @@
+#!/usr/bin/env python3
```

Comment:
This is a great addition. But would you make another PR which addresses the missing integration test?